### DISCUSSION TITLE
blog/2015/redsocks2/

### DISCUSSION THREAD

**Discussion by giscus** [go-gost/gost#563] [General]
# blog/2015/redsocks2/

https://gost.run/blog/2015/redsocks2/

**Comment by Kingxukai** [comment]
您好。感谢的你的教程。
我已经执行完“redsocks -c redsocks.conf”这一步，但是之后我的计算机网络代理可能出现了点问题。
我想知道这一步执行完修改了哪些配置呢？感谢您的回答
